yarpl/test/FlowableTest.cpp

```diff
@@ -538,5 +540,58 @@ TEST(FlowableTest, FlowableFromDifferentThreadsWithError) {
 }
 } // namespace
 
+TEST(FlowableTest, SubscribeMultipleTimes) {
+  using namespace ::testing;
+  using StrictMockSubscriber = testing::StrictMock<yarpl::mocks::MockSubscriber<int64_t>>;
+  auto f = Flowable<int64_t>::create([](auto subscriber, int64_t req) {
+    for(int64_t i = 0; i < req; i++) {
+      subscriber->onNext(i);
+    }
+
+    subscriber->onComplete();
+    return std::make_tuple(req, true);
+  });
+
+  auto setup_mock = [](auto request_num, auto& resps) {
+    auto mock = make_ref<StrictMockSubscriber>(request_num);
+
+    Sequence seq;
+    EXPECT_CALL(*mock, onSubscribe_(_)).InSequence(seq);
+    EXPECT_CALL(*mock, onNext_(_))
+      .InSequence(seq)
+      .WillRepeatedly(Invoke([r = &resps](int64_t value) {
```

Comment:
Nit: Why not just `[&resps]`?

Reply:
good call; prior to this resps was on the stack